yt/yql/agent/config.cpp

```diff
@@ -214,6 +214,8 @@ void TDqManagerConfig::Register(TRegistrar registrar)
         .Default(4);
     registrar.Parameter("use_ipv4", &TThis::UseIPv4)
```

Comment:
Or, we can remove it and use only the AddressResolver config

Reply:
AddressResolver allows both ipv4 and ipv6. [This code](https://github.com/ydb-platform/ydb/blob/ee419dbcdec5f5bac819a17618598ca2cf13e4c2/ydb/library/yql/tools/dq/worker_job/dq_worker.cpp#L203-L206) can work only with one of them

We will need to rewrite it to work with both ipv4 and ipv6 in this case

Reply:
Does this PR changes anything in this case?

I suppose that it starts working with complex (ipv4+ipv6) clusters. And I mean that in case of existence address_resolver this option means that we force IPv4 using for Coordination Helper address, but we allow to use both types of addresses for resolving

Reply:
>Does this PR changes anything in this case?

Yes. It starts working on a cluster described in [the issue](https://github.com/ytsaurus/ytsaurus/issues/744)

I can't locate in the code why it does

Here are the details how the cluster behaved:
- old code. useIpv4 = false: `Ares DNS resolve failed for "http-proxies.our-cluster.svc.cluster.local" DNS server returned answer with no data`
- old code. useIpv4 = true: `Ares DNS resolve failed for "dp-a-8.yt.our-cluster.com" DNS server returned answer with no data`
- new code. useIpv4 = false: All working and good
- new code. useIpv4 = true: DQ fails to start with: https://github.com/ydb-platform/ydb/issues/7296